src/DotNetWorker.Core/Context/DefaultFunctionContext.cs

```diff
@@ -12,13 +13,15 @@ internal sealed class DefaultFunctionContext : FunctionContext, IAsyncDisposable
     {
         private readonly IServiceScopeFactory _serviceScopeFactory;
         private readonly FunctionInvocation _invocation;
+        private readonly CancellationToken _cancellationToken;
 
         private IServiceScope? _instanceServicesScope;
         private IServiceProvider? _instanceServices;
         private BindingContext? _bindingContext;
 
-        public DefaultFunctionContext(IServiceScopeFactory serviceScopeFactory, IInvocationFeatures features)
+        public DefaultFunctionContext(IServiceScopeFactory serviceScopeFactory, IInvocationFeatures features, CancellationToken cancellationToken)
```

Comment:
I like to always have `= default` for all cancellation arguments. It is nice to omit it when you don't care about cancellation, IE: unit tests.